src/run.ts

```diff
@@ -286,27 +289,29 @@ ${
     projectId: context.projectId,
     state: 'opened',
     sourceBranch: versionBranch,
-    target_branch: branch,
+    target_branch: mrTargetBranch,
     maxPages: 1,
     perPage: 1,
   })
   console.log(JSON.stringify(searchResult, null, 2))
   if (searchResult.length === 0) {
-    console.log('creating merge request')
+    console.log(
+      `Creating merge request from ${versionBranch} to ${mrTargetBranch}.`,
+    )
     await api.MergeRequests.create(
       context.projectId,
       versionBranch,
-      branch,
+      mrTargetBranch,
       finalMrTitle,
       {
         description: await mrBodyPromise,
       },
     )
   } else {
+    console.log('Found existing merge request, updating.')
```

Comment:
Why this change is needed?

Reply:
consistency

Reply:
Or align

https://github.com/changesets/action/blob/398d7edfb130ad2d4e076a573724f435ea87cea6/src/run.ts#L359

Besides, can you un-capitalize the messages for consistency with upstream?

Reply:
done!